travel_platform/travel_app/views.py

```diff
@@ -1,12 +1,33 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
-
-from django.http import Http404
-
 from django.shortcuts import render
+from django.views.decorators.cache import cache_page
 
-from .models import Query
+from api.izi_travel.izi_travel_data import *
 
 
 def index(request):
     return render(request, 'travel_app/main.html')
+
+
+@cache_page(86400)
```

Comment:
It would be great to have TTL as configuration parameter instead of hard-coding.